sdk/src/manifest.rs

```diff
@@ -1087,6 +1083,45 @@ pub(crate) mod tests {
         assert!(claim1.get_claim_assertion(redacted_uri, 0).is_none());
     }
 
+    #[test]
+    #[cfg(feature = "file_io")]
+    fn test_action_assertion_not_retractable() {
+        let temp_dir = tempdir().expect("temp dir");
+        let parent_output = temp_fixture_path(&temp_dir, TEST_SMALL_JPEG);
+
+        // Create parent with a c2pa_action type assertion.
+        let mut parent_manifest = test_manifest();
+        let actions = Actions::new().add_action(
+            Action::new(c2pa_action::FILTERED)
+                .set_parameter("name".to_owned(), "gaussian blur")
+                .unwrap()
+                .set_when("2015-06-26T16:43:23+0200"),
+        );
+        parent_manifest
+            .add_assertion(&actions)
+            .expect("add_assertion");
+
+        let signer = temp_signer();
+        parent_manifest
+            .embed(&parent_output, &parent_output, &signer)
+            .expect("embed");
+
+        // Add parent_manifest as an ingredient of the new manifest and redact the assertion `c2pa.actions`.
+        let mut manifest = test_manifest();
+        manifest
+            .set_parent(Ingredient::from_file(&parent_output).expect("from_file"))
+            .expect("set_parent");
+        assert!(manifest.add_redaction(ACTIONS).is_ok());
+
+        // Attempt embedding the manifest with the invalid retraction.
+        let retract_output = temp_fixture_path(&temp_dir, TEST_SMALL_JPEG);
```

Comment:
retract or redact? just curious about the alternate terms

Reply:
My mistake, will fix before merging 👍